uint8[] PingerIDs
---
sonar_node/PingerArray PingerArray
